csfieldguide/chapters/management/commands/_GlossaryTermsLoader.py

```diff
@@ -20,16 +20,13 @@ def __init__(self, structure_file_path, glossary_directory_name, BASE_PATH):
             BASE_PATH (str): Base file path.
         """
         super().__init__(BASE_PATH)
-        print(self.BASE_PATH, glossary_directory_name)
         self.structure_file_path = structure_file_path
         self.glossary_folder_path = os.path.join(self.BASE_PATH, glossary_directory_name)
         self.FILE_EXTENSION = ".md"
 
     @transaction.atomic
     def load(self):
         """Load the glossary content into the database."""
-        # glossary_slugs = set()
-
         for filename in listdir(self.glossary_folder_path):
             if filename.endswith(self.FILE_EXTENSION):
```

Comment:
Files should require `.md` extension. There is no other files that have this option.